src/UCTNode.cpp

```diff
@@ -313,6 +343,17 @@ class NodeComp : public std::binary_function<UCTNodePointer&,
         auto a_visit = a.get_visits();
         auto b_visit = b.get_visits();
 
+        // Calculate the lower confidence bound for each node.
+        if (a_visit && b_visit) {
+            auto a_lcb = a.get_lcb(m_color);
+            auto b_lcb = b.get_lcb(m_color);
```

Comment:
Similarly, does it matter that the number of visits used to calculate the lcbs could be different to a_visits/b_visit?
I know we’ve had a similar race condition issue here previously.

Reply:
I don't think it's an issue. a_visit and b_visit variables are just checked that they are both bigger than zero. lcb check isn't really any different from other checks in the function.